.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
```

Comment:
@oliver-nyt happy to remove, but i think projects tend to set this to prevent configurations higher up in the filesystem tree (e.g, user settings) from resulting in auto generated changes that may not be desired within the project.. here are a few selected examples from the projects listed [here](https://github.com/editorconfig/editorconfig/wiki/Projects-Using-EditorConfig):

[`htop`](https://github.com/hishamhm/htop/blob/master/.editorconfig)
[`qemu`](https://git.qemu.org/?p=qemu.git;a=blob;f=.editorconfig;h=b2022e391abfeaf4d7a4eb9b14e682ebf7ba8a43;hb=refs/heads/master)
[SpringBoot](https://github.com/spring-projects/spring-boot/blob/master/.editorconfig)

wdyt? keep or remove?

Reply:
Your call. I personally don't like checking in user tools specific stuff like editor configs (this is for VSCode only, right?) but I see your point, it makes life of some users easier. Interesting that so many projects do it.

Reply:
editorconfig isn't specific to vscode: https://editorconfig.org/#download :grin:

Reply:
<img src="https://media0.giphy.com/media/3og0IMJcSI8p6hYQXS/giphy.gif">